cogeo_mosaic/mosaic.py

```diff
@@ -72,20 +72,32 @@ class MosaicJSON(BaseModel):
     minzoom: int = Field(0, ge=0, le=30)
     maxzoom: int = Field(30, ge=0, le=30)
     quadkey_zoom: Optional[int]
-    bounds: Tuple[float, float, float, float] = Field((-180, -90, 180, 90))
+    bounds: Tuple[float, float, float, float] = Field(default=(-180, -90, 180, 90))
     center: Optional[Tuple[float, float, int]]
     tiles: Dict[str, List[str]]
-
-    # TMS is a private attributes for now but it might change with
-    # mosaicJSON 0.3.0 (https://github.com/developmentseed/mosaicjson-spec/pull/18)
-    # Note: TMS should support quadkey (tms._is_quadtree)
-    _tms: morecantile.TileMatrixSet = PrivateAttr(default=WEB_MERCATOR_TMS)
+    tilematrixset: Optional[morecantile.TileMatrixSet]
```

Comment:
should we set a default?

Reply:
I think it's fine to assume that all mosaicJSON without tilematrixset attribute default to WebMercatorQuad 🤷